### PR TITLE
epiv2 data pivots and exploratory heatmaps

### PR DESCRIPTION
**Pull request by shapiromatron**
Add the ability to create data pivots and exploratory heatmaps using data extracted in the epiv2 module.    

* We used the basic data export for data extraction data, and created a simple systematic evidence map (SEM) style study summary visual. It's likely that these exports will change in the future, and we'll need to make sure existing visuals are migrated
* Added an outcome list view for epiv2 data s well
* Added some API test helper utility methods to make testing datasets simpler

A future PR will improve the default exploratory heatmaps which are created on the "endpoint list" page. This PR implements the initial versions.

### CODE REVIEW

File: hawc/apps/common/models.py [hunk 1]
```diff
@@ -518,6 +518,54 @@ def include_related(
...
+    """Create a annotation to return the display name via SQL
+
+    Args:
+        name (str): the field name
+        Choice (type[Choices]): a choice field
+
+    Returns:
+        Case: the case statement for use in an annotation
+    """
+    return Case(
```
Comment (shapiromatron) [inline hawc/apps/common/models.py:531]: @casey1173

File: hawc/apps/assessment/views.py [hunk 2]
```diff
@@ -695,12 +695,18 @@ def get_context_data(self, **kwargs):
         iveps = self.model.ivendpoint.related.related_model.objects.get_qs(
             self.assessment.id
         ).count()
-        alleps = eps + os + mrs + iveps
+        epiv2_designs = apps.get_model("epiv2", "Design").objects.get_qs(self.assessment.id).count()
+        epiv2_data_extractions = (
```
Comment (casey1173) [inline hawc/apps/assessment/views.py:699]: this is counting epiv2 data extractions, but it's a list of epiv2 outcomes

Comment (casey1173) [reply hawc/apps/assessment/views.py:699]: @shapiromatron

Comment (shapiromatron) [reply hawc/apps/assessment/views.py:699]: Thanks, that seems off, I will fix

**Review by casey1173 — APPROVED** [review]

**Pull request merged**